process_manager/src/ProcessManager.cpp

```diff
@@ -117,7 +117,7 @@ void ProcessManager::handleProcessCommand(process_manager::ProcessCommandPtr pc)
     // check whether this message is for me, 0 is a wild card for all ProcessManagers
     const AgentID* receiverId;
     if (pc->receiverId.type == supplementary::AgentID::BC_TYPE) {
-        receiverId = new supplementary::BroadcastID(nullptr, 0);
+        receiverId = supplementary::BroadcastID(nullptr, 0);
```

Comment:
This doesn't seem correct and maybe this PR can be declined now.

Reply:
Yes, it is not correct. @abolinsky please eithe remove the pr or make a proper solution ;)